### ISSUE TITLE
Query error when inserting records in a table via ysql

### ISSUE THREAD

**Issue by chris-ng-scmp** [yugabyte/yugabyte-db#2500]
I have followed the document here https://docs.yugabyte.com/latest/quick-start/explore-ysql/ to test Yugabyte YSQL, I have 3 tserver and 3 master servers.

When I running `\i 'data/orders.sql'`, and then run `select count(*) from orders;` while the data importing into the table following error will show until the import is done.

```
yb_demo=# select * from orders;
select * from orders;
ERROR:  Query error: Restart read required at: { read: { physical: 1570069554431110 } local_limit: { physical: 1570069554477813 } global_limit: <min> in_txn_limit: <max> serial_no: 0 }
```

With this behaviour, I assume I can't use Yugabyte for production env.

**Comment by kmuthukk** [comment]
hi @chris-ng-scmp -- yes, this is an important issue we are working on right now to unblock soon (to automatically handle read restarts if there are concurrent writes happening to same rows).

See https://github.com/yugabyte/yugabyte-db/issues/2216

Closing this one as a duplicate of #2216.
